### PR TITLE
save current user to session

### PR DESCRIPTION
**Pull request by BernardTolosajr**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -2,9 +2,14 @@ import Ember from 'ember';
 import Resolver from 'ember/resolver';
 import loadInitializers from 'ember/load-initializers';
 import config from './config/environment';
+import fetch from 'fetch';
 
 var App;
 
+// HACK: this is to override the native fetch with ember-fetch.
+// ember-fetch play nicely with pretender
+window.fetch = fetch;
```
Comment (oliverbarnes) [inline app/app.js:11]: Nice, but is there a way to use a initializer for this? And make it restricted to the test environment

File: app/sessions/me.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import Session from 'simple-auth/session';
+
+export default Session.extend({
+  me: function() {
+    var accessToken = this.get('secure.access_token');
+    if (!Ember.isEmpty(accessToken)) {
+      let self = this;
+      return this.container.lookup('service:me').find({}).then((me) => {
```
Comment (oliverbarnes) [inline app/sessions/me.js:9]: :+1:  :+1: man I spent so much time on this. much appreciated

File: tests/acceptance/current-user-test.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    visit('/');
+  });
+
+  afterEach(function() {
+    server.shutdown();
+    Ember.run(application, 'destroy');
+  });
+
+  it('display current user', function() {
```
Comment (oliverbarnes) [inline tests/acceptance/current-user-test.js:49]: Very nice, thanks dude. I'm merging this into the working branch and then merging everything into master. Please submit another PR for the `window.fetch = fetch` initializer?

**Comment by oliverbarnes** [comment]
Btw, going to add you as a collaborator on the main repo :)

File: tests/acceptance/current-user-test.js (hunk 3 above)
Comment (BernardTolosajr) [inline tests/acceptance/current-user-test.js:49]: Sure thing, i'll try to implement the window.fetch with the initializer.
